openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -312,6 +318,42 @@ def list(self, request):
 
         return Response(account_settings)
 
+    def search_emails(self, request):
+        """
+        POST /api/user/v1/accounts/search_emails
+        Content Type: "application/merge-patch+json"
+        {
+            "emails": ["edx@example.com", "staff@example.com"]
+        }
+        Response:
+        [
+            {
+                "username": "edx",
+                "email": "edx@example.com",
+                "id": 3,
+            },
+            {
+                "username": "staff",
+                "email": "staff@example.com",
+                "id": 8,
+            }
+        ]
+        """
+        try:
+            user_emails = request.data['emails']
+        except KeyError as err:
+            mesg = f'{err} field is required'
+            return Response(
+                {
+                    'developer_message': mesg,
+                    'user_message': mesg
+                },
+                status=status.HTTP_400_BAD_REQUEST
+            )
+        users = User.objects.filter(email__in=user_emails)
+        data = UserSearchEmailSerializer(users, many=True).data
```

Comment:
We also need to handle the scenario when no users are returned in search

Reply:
it will return an empty list.